at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -712,6 +727,9 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       if (metaData[SHARED_WITH_PUBLIC_KEY_CHECK_SUM] != null) {
         builder.pubKeyChecksum = metaData[SHARED_WITH_PUBLIC_KEY_CHECK_SUM];
       }
+      if (metaData['encoding'] != null) {
```

Comment:
replace all use of literal `'encoding'` with a constant, declared in the same place as `SHARED_KEY_ENCRYPTED` etc

Reply:
Replaced with constant.